components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/PublisherCommonUtils.java

```diff
@@ -1093,7 +1093,6 @@ public static String updateSwagger(String apiId, APIDefinitionValidationResponse
 
         existingAPI.setUriTemplates(uriTemplates);
         existingAPI.setScopes(scopes);
-        existingAPI.setOrganization(organization);
```

Comment:
Let's keep this setOrganization() call and remove the one in line 1105. This line is needed for validateScopes method.